.azure/test-linux.yml

```diff
@@ -50,6 +51,12 @@ jobs:
           rustup default "${{ parameters.rustVersion }}"
         displayName: "Use rust version ${{ parameters.rustVersion }}"
 
+      - ${{ if eq(parameters.testRust, true) }}:
+        - bash: |
+            set -e
+            cargo test --tests -- --nocapture
```

Comment:
I copied this from Pantsbuild's CI. 

* The `--tests` says to only run `--tests` and skip Rust examples and benchmarks. We don't have either of those afaict, so this might be premature to add
* The `--nocapture` has been useful to us debugging tests because std{out,err} will show up on failed tests

Reply:
The Python runners (`pytest` and `stestr` for us) both capture by default, and then show the captured output in the result for that test iff it failed.  I don't know about others, but that's my preferred behaviour if it's available with `cargo test`.  Probably not relevant here, but it's especially important when running tests in parallel.

Reply:
Hm yeah I re-read things and `--nocapture` results in printing things like `println!()` even for successful tests. So I removed this